tools/small_rna_maps/small_rna_maps.r

```diff
@@ -220,9 +220,9 @@ bottom_extra_method =list(Counts="Coordinates (nbre of bases)",Coverage="Coordin
 ## Plotting Functions
 
 double_plot <- function(...) {
-    if (n_genes > 5) {page_height=15; rows_per_page=10} else {
-                     rows_per_page= 2 * n_genes; page_height=1.5*n_genes}
-    if (n_samples > 4) {page_width = 8.2677*n_samples/4} else {page_width = 7 * n_samples/2}
+    if (n_genes > 5) {page_height=15; rows_per_page=10; graph_heights=c(40,30,40,30,40,30,40,30,40,30,10)} else {
+                     rows_per_page= 2 * n_genes; page_height=1.5*n_genes + 4; graph_heights=c(rep(c(40,30),n_genes), '10')}
+    if (n_samples > 4) {page_width = 8.2677*n_samples/4} else {page_width = 7}
```

Comment:
are you sure of `else {page_width = 7}` ?
I am concerned by "stretched" graphs when n_samples = 2 or 1